sending-email.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Sending email
+layout: recommendation
+expires: 2017-11-01
+---
+
+
+## User needs for sending email
+
+- applications notifying users about progress or actions to take
+- automated notifications sent to engineers by systems such as cron or jenkins remembering that we [don’t rely on emails as an automated alerting method for incident responders](https://alphagov.github.io/gds-tech/alerting.html#principles-for-setting-up-an-alerting-system)
```

Comment:
yes they should, however not everything gets caught like that. they were examples of things which may get sent via email. 

For instance gov.uk integration deploys arnt easily monitored and people dont notice failed deploys.